### DISCUSSION TITLE
Add support for Revolut Pay

### DISCUSSION THREAD

**Discussion by cortopy** [juspay/hyperswitch#5110] [Ideas / Feature Requests]
It seems that [Revolut Pay](https://www.revolut.com/business/revolut-pay/) is not supported as a connector. 

I’m just opening this PR to request integration if possible

**Comment by gorakhnathy7** [comment]
Hey @cortopy, thanks for this feature request.

As you rightly pointed out, we currently do not have Revolut Pay with us. However, this can be discussed internally and added to our Roadmap.

Can you elaborate on the use-case for which Revolut Pay is required?

**Comment by cortopy** [comment]
Hi @gorakhnathy7 , thanks for replying to my feature request.

My business case is being able to pay less for online payments. [Revolut fees](https://www.revolut.com/business/accept-payments-pricing/) are cheaper than Stripe's, which is what we're currently using. However, Revolut does not accept some payment methods like American Express. 

My use case would be to use Hyperswitch to use the cheapest option (in this case Revolut) as often as possible (e.g.: Visa, Mastercard) and use Stripe for the rest (e.g.: American Express).

Revolut is available all over Europe for business accounts I bleieve

**Reply by gorakhnathy7** [reply]
Thanks, we will keep you posted with the updates.
